components/x-podcast-launchers/src/PodcastLaunchers.jsx

```diff
@@ -48,39 +48,38 @@ class PodcastLaunchers extends Component {
 
 	render() {
 		const { rssUrl } = this.state;
-		const { conceptId, conceptName, csrfToken, isFollowed, renderFollowButton } = this.props
+		const { conceptId, conceptName, csrfToken, isFollowed, renderFollowButton, showLinksOnAllBreakpoints = true } = this.props
 		const followButton = typeof renderFollowButton === 'function' ? renderFollowButton : defaultFollowButtonRender;
 
 		return rssUrl && (
 			<div className={styles.container} data-trackable='podcast-launchers'>
 				<h2 className={styles.headingChooseApp}>Subscribe via your installed podcast app</h2>
-				<ul className={styles.podcastAppLinksWrapper}>
-					{generateAppLinks(rssUrl).map(({ name, url, trackingId }) => (
-						<li key={name}>
-							<a
-								href={url}
-								className={styles.podcastAppLink}
-								data-trackable={trackingId}>
-								{name}
-							</a>
-						</li>
-					))}
-
-					<li key='Rss Url' className={styles.rssUrlWrapper}>
-						<span className={rssUrlWrapperInner}>
-							<input value={rssUrl} type='text' readOnly/>
-								<button
-									className={styles.rssUrlCopyButton}
-									onClick={copyToClipboard}
-									data-url={rssUrl}
-									data-trackable='copy-rss'
-									type='button'>
-									Copy RSS
-								</button>
-						</span>
-					</li>
-				</ul>
+					<ul className={styles.podcastAppLinksWrapper}>
+						{generateAppLinks(rssUrl).map(({ name, url, trackingId }) => (
+							<li key={name} className={showLinksOnAllBreakpoints ? "" : styles.hidePodcastLinkAtWide}>
```

Comment:
it would be better to output `null` instead of empty string here. empty string would output as `class=""` whereas `null` would omit the attribute.

Reply:
```suggestion
							<li key={name} className={showLinksOnAllBreakpoints ? null : styles.hidePodcastLinkAtWide}>
```